apps/src/templates/RotateContainer.jsx

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var msg = require('../locale');
+
+/**
+ * "Rotate your device" overlay.
+ */
+var RotateContainer = React.createClass({
+  propTypes: {
+    assetUrl: React.PropTypes.func.isRequired
+  },
+
+  render: function () {
+    return (
+      <div id="rotateContainer" style={this.getStyle()}>
+        <div id="rotateText">
+          <p>{ msg.rotateText() }<br />{msg.orientationLock()}</p>
```

Comment:
Any reason for the whitespace surrounding msg.rotateText() but not msg.orientationLock?